release-tools/scripts/required_packages.sh

```diff
@@ -2,9 +2,9 @@
 
 ARCH=`uname -p | tr '[:upper:]' '[:lower:]'`; echo $ARCH
 PLATFORM=`uname -s | tr '[:upper:]' '[:lower:]'`; echo $PLATFORM
-DEB_PKGS="curl wget unzip tar jq python python3 git awscli libnss3-dev fonts-liberation libfontconfig1"
+DEB_PKGS="curl wget unzip tar jq python python3 git awscli libnss3-dev fonts-liberation libfontconfig1 python-setuptools"
 RPM_PKGS="curl wget unzip tar jq python python3 git awscli libnss3.so xorg-x11-fonts-100dpi xorg-x11-fonts-75dpi \
-          xorg-x11-utils xorg-x11-fonts-cyrillic xorg-x11-fonts-Type1 xorg-x11-fonts-misc fontconfig freetype"
+          xorg-x11-utils xorg-x11-fonts-cyrillic xorg-x11-fonts-Type1 xorg-x11-fonts-misc fontconfig freetype python-setuptools"
```

Comment:
Did you check if yum actually has python-setuptools?

Reply:
Yes. yum has python-setuptools